File: pde_matlab/A1.m
function y = A1(r,a, kr, kL, B, b, kt, v0, vl, na, L0)

    v = 4 / 3 * pi * r .^3;
    
    Tau_r = (a /(4*pi) * 1./(1 + (r/kr).^na) * L0/(kL+L0)) - (B + b * r.^2)./(4*pi * r.^2) .* (v - v0)./(v - v0 + kt*vl);
    y = vl * Tau_r;
end

 
 
 

File: pde_python/A1.py
# -*- coding: utf-8 -*-
"""
Created on Wed Jun 26 10:39:36 2019
@author: Léo Meyer

"""

import numpy as np

#====================================================

# Calcul de la fonction de transport

def TauR(r, a, r_theta, nr, T_theta, B, b, l_theta, v0, vl, T):
    
	v = 4/3. * np.pi * r**3	
	Tau_r = (a /(4*np.pi) * 1./(1 + (r/r_theta)**nr) * T/(T_theta+T)) - (B + b * r**2)/(4*np.pi * r**2) * (v - v0)/(v - v0 + l_theta*vl)
	
	y = vl * Tau_r
	
	return y
	
#====================================================
